### PR TITLE
addition of multi-set & multi-remove

### PR DESCRIPTION
**Pull request by derHowie**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ pom.xml.asc
 /.nrepl-port
 .cljs_rhino_repl
 node_modules
+*.swp
```
Comment (glittershark) [inline .gitignore]: I generally don't like to pollute repo-specific .gitignores with editor or tool-specific items. This can go in a global gitignore

**Review by glittershark — CHANGES_REQUESTED** [review]
Like the API quite a bit overall. Would you mind putting this all in one commit, though, in addition to removing the swap file ignore?

Thanks for the PR!

**Comment by derHowie** [comment]
No problem, I squashed the commits and restored the .gitignore file.

**Comment by derHowie** [comment]
@glittershark, this PR now includes multi-remove as well

**Comment by glittershark** [comment]
Thanks! Will push out a version soon.

**Pull request merged**